joint_limits:
  base_rotation_joint:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
  shoulder_pitch_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
  elbow_pitch_joint:
    has_position_limits: true
    min_position: -2.0
    max_position: 2.0
  wrist_pitch_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
  wrist_roll_joint:
    has_position_limits: true
    min_position: -3.14
    max_position: 3.14
